### PR TITLE
ETV-2672-sidebar-header

### PR DESCRIPTION
**Pull request by sergiuelisei**
https://etvas.atlassian.net/browse/ETV-2672

### CODE REVIEW

File: src/Layout/AppLayout.jsx [hunk 1]
```diff
@@ -229,6 +235,7 @@ AppLayout.propTypes = {
   footerContent: PropTypes.node,
   footerTooltip: PropTypes.node,
   footerBgImage: PropTypes.node,
+  headerPadding: PropTypes.oneOfType(PropTypes.string, PropTypes.array),
```
Comment (alsofronie) [inline src/Layout/AppLayout.jsx:238]: Can be also a number?

```suggestion
  headerPadding: PropTypes.oneOfType(PropTypes.string, PropTypes.array, PropTypes.number),
```

File: src/Layout/AppLayout.jsx [hunk 2]
```diff
@@ -238,5 +245,6 @@ AppLayout.propTypes = {
 }
 
 AppLayout.defaultProps = {
-  variant: ['mobile', null, 'web']
+  variant: ['mobile', null, 'web'],
+  headerPadding: '24px'
```
Comment (alsofronie) [inline src/Layout/AppLayout.jsx:249]: Can't this be:

```suggestion
  headerPadding: 6
```

**Review by alsofronie — CHANGES_REQUESTED** [review]

File: src/Layout/AppLayout.jsx (hunk 2 above)
Comment (sergiuelisei) [reply src/Layout/AppLayout.jsx]: ✅

File: src/Layout/AppLayout.jsx (hunk 1 above)
Comment (sergiuelisei) [reply src/Layout/AppLayout.jsx]: ✅

**Review by alsofronie — APPROVED** [review]

**Pull request merged**